packages/jsonforms-renderers/src/renderers/DependentSelectControl.tsx

```
@@ -0,0 +1,117 @@
+    props: ownProps,
+  }: {
+    ctx: JsonFormsStateContext
+    props: OwnPropsOfControl
+  }) {
+    const controlProps = ctxToControlProps(ctx, ownProps)
+    // dispatch is always set once JsonForms has initialized — this control never renders before that
+    const dispatchProps = ctxDispatchToControlProps(ctx.dispatch!)
+    return <Component {...ownProps} {...controlProps} {...dispatchProps} rootData={ctx.core?.data} />
+  })
+
+export default withDependentSelectProps(DependentSelectControl)
```

ESLint

[error] Fast refresh can't handle anonymous components. Add a name to your export.

(react-refresh/only-export-components)